<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_XS_ADD</class>
 <widget class="QDialog" name="Dlg_XS_ADD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_XS">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="txt_XS_SN">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>交易时间</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>金      额</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>商品品牌</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="txt_XS_brand">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QLineEdit" name="txt_XS_unit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2">
       <widget class="QLineEdit" name="txt_XS_type">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>商品型号</string>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QLineEdit" name="txt_XS_jine">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>单     位</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>数      量</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txt_XS_cnt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>交易序号</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>备      注</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>支付方式</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>现金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>微信</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>支付宝</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4" colspan="5">
       <widget class="QLineEdit" name="txt_XS_jytime">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="4">
       <widget class="QLineEdit" name="txt_XS_memo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QLineEdit" name="txt_XS_price">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>单     价</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dlg_XS_ADD</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dlg_XS_ADD</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
